docker/standalone/docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
       - indexer
     restart: always
   indexer: &ortelius-app
-    image: "avaplatform/ortelius:v1.6.1"
+    image: "avaplatform/ortelius:v1.6.2"
```

Comment:
Why is this 1.6.2 and not 1.6.3 or 1.6.4?
I don't know what this is actually doing, but the version caught my eye

Reply:
I can update this to 1.6.4.  This is the Ortelius version, which now that I'm looking at it looks like it was in sync with AvalancheGo, but fell behind

Reply:
Actually on further inspection there isn't an Ortelius v1.6.4 tag, which I assume breaks this